at-hoangtran/Week3/Day5/Exercise2-Day5.rb

```diff
@@ -0,0 +1,57 @@
+class Dictionary
+  attr_accessor :dictionary, :vocabulary
+  def initialize
+    @dictionary = []
+    @vocabulary = []
+  end
+
+  def add(vocab, desc)
+    dict = {}
+    unless @vocabulary.include? (vocab)
+      dict[vocab] = desc
+      @vocabulary.push(vocab)
+      @dictionary.push(dict)
+    end
+  end
+  def lookup(vocab)
+    if @vocabulary.include? (vocab)
+      return dict[vocab]
+    else
+      return nil
+    end
+  end
+  def size
+    @dictionary.size
+  end
+  def random
+    @dictionary.sample
+  end
+  def size
+    @dictionary.size
+  end
+  def remove(vocab)
+    if @vocabulary.include?(vocab)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/2bfb283d73880abdd7ab3691785775673a31497a/at-hoangtran/Week3/Day5/Exercise2-Day5.rb#L33"></span>Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. (https://github.com/rubocop-hq/ruby-style-guide#no-nested-conditionals)</td>
    </tr>
  </tbody>
</table>